{
  try {
    executePreparedStatement("insert into TestPACL_InsertFailure values (1)");
    Assert.fail();
  }
 catch (  SecurityException se) {
  }
}
